src/mlpack/methods/hdbscan/hdbscan_main.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file hdbscan_main.cpp
+ * @author Sudhanshu Ranjan
+ *
+ * Implementation of program to run HDBSCAN.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+
+#include "hdbscan.hpp"
+
+using namespace mlpack;
+using namespace mlpack::hdbscan;
+using namespace std;
+
+PROGRAM_INFO("HDBSCAN algorithm", "This program helps in clustering "
```

Comment:
I think we should make this documentation a lot longer and add some example usage.  I can help with this part, and we can do it when we get closer to merge.